mimic/rest/nova_api.py

```diff
@@ -315,18 +325,16 @@ def delete_server(self, request, tenant_id, server_id):
         """
         Returns a 204 response code, for any server id'
         """
-        return (
-            self._region_collection_for_tenant(tenant_id)
-            .request_delete(request, server_id)
-        )
+        return (self._region_collection_for_tenant(tenant_id)
+                .request_delete(request, server_id))
 
     @app.route('/v2/<string:tenant_id>/images/<string:image_id>', methods=['GET'])
     def get_image(self, request, tenant_id, image_id):
         """
         Returns a get image response, for any given imageid
         """
-        return(self._image_collection_for_tenant(tenant_id)
-               .get_image(request, image_id, absolutize_url=self.url))
+        return (self._image_collection_for_tenant(tenant_id)
```

Comment:
I try to review each line carefully.  This actually makes cleanups that don't change anything else particularly hard to review, because I have to first realize that nothing has changed, then verify that nothing has changed.  It adds to the effort of each review.  Furthermore, it pollutes change history, by making it harder to read `git annotate` and diagnose why a certain line is a certain way.  Although this is a content-free reformat, it will appear alongside a merge commit message that suggests something more substantive is changing.  Given that Mimic has relatively light maintenance documentation and is in a constant state of evolving towards a better API, `annotate` is a particularly important and powerful tool for understanding how things got to where they are.

This doesn't mean we should never do cleanups, but rather, cleanup branches should be their own small changes.

Since you are in a bit of a time crunch here I can build on top of your PR and resubmit for review myself though.